core/MyGatewayTransportSerial.cpp

```diff
@@ -42,7 +42,7 @@ bool gatewayTransportSend(MyMessage &message) {
 bool gatewayTransportInit() {
 	gatewayTransportSend(buildGw(_msgTmp, I_GATEWAY_READY).set("Gateway startup complete."));
 	// Send presentation of locally attached sensors (and node if applicable)
-	presentNode();
+	// presentNode();
```

Comment:
Ups, shouldn't be commented - this will be part of a different PR :) 
Thanks for reporting.